### ISSUE TITLE
Usage of deprecated OpenVINO packages under Python 3.12 prevents using OpenVINO models

### ISSUE THREAD

**Issue by scudo005** [ggml-org/whisper.cpp#2190]
If I try to install the required packages in order to use OpenVINO, the build fails.
I opened an issue on the OpenVino repository [(here)](https://github.com/openvinotoolkit/openvino/issues/24686), and I got told that the package `openvino-dev`, which is present in requirements-openvino.txt.
From my own observations, said package seems to be available only on Python 3.11 and not 3.12.

**Comment by recallmenot** [comment]
yes, currently there appears to be no way to convert models to openvino due to python 3.12
@DocSwitch did you ever figure out how to use openvinos ovt to accomplish this?

**Comment by adrian-heouairi** [comment]
I just successfully generated a .srt file with a Python 3.12 on Lubuntu 24.04, went from 272 seconds to 127 seconds (2.14 times faster) for a 189 seconds .wav music on Dell XPS 13 9310 laptop with 11th Gen Intel(R) Core(TM) i7-1185G7 with medium (not medium en) model
